src/index.css

```diff
@@ -88,6 +88,10 @@ body {
   color: var(--formal-accent);
 }
 
+.stroke-text-secondary {
+  -webkit-text-stroke: 2px var(--secondary);
```

Comment:
Supported by 99% of browsers, including FF: https://caniuse.com/?search=-webkit-text-stroke